### PR TITLE
Bring acoustic settings to basket-stage

### PR DESCRIPTION
**Pull request by robhudson**

### CODE REVIEW

File: mozmeao-fr/basket-stage/web-deploy.yaml [hunk 1]
```diff
@@ -38,6 +38,42 @@ spec:
...
+        - name: ACOUSTIC_CLIENT_SECRET
+          valueFrom:
+            secretKeyRef:
+              key: acoustic-client-secret
+              name: basket-prod-secrets
+        - name: ACOUSTIC_REFRESH_TOKEN
+          valueFrom:
+            secretKeyRef:
+              key: acoustic-refresh-token
+              name: basket-prod-secrets
```
Comment (pmac) [inline mozmeao-fr/basket-stage/web-deploy.yaml]: these should all be `basket-stage-secrets`

File: mozmeao-fr/basket-stage/worker-deploy.yaml [hunk 2]
```diff
@@ -38,6 +38,42 @@ spec:
         - /bin/bash
         - -c
         env:
+        - name: ACOUSTIC_CLIENT_ID
+          valueFrom:
+            secretKeyRef:
+              key: acoustic-client-id
+              name: basket-prod-secrets
```
Comment (pmac) [inline mozmeao-fr/basket-stage/worker-deploy.yaml]: `basket-stage-secrets`

File: mozmeao-fr/basket-stage/web-deploy.yaml (hunk 1 above)
Comment (robhudson) [reply mozmeao-fr/basket-stage/web-deploy.yaml]: Okay. I wasn't sure the preference here. I can see liking the visibility that these settings are the same as prod. But it is crossing the streams a bit.

Comment (pmac) [reply mozmeao-fr/basket-stage/web-deploy.yaml]: I believe it simply won't work. The secrets are added to their respective namespaces, so configs in the stage namespace won't have access to secrets in the prod one.

**Review by pmac — APPROVED** [review]
🔒

**Pull request merged**